README.md

```diff
@@ -1 +1,72 @@
-# .github
\ No newline at end of file
+
+## Repositories: 
+- Service
+- Purpose
+- Language
+- External Integration
+- Database
+- Infrastructure
+- Main contributor
+
+### 1. ubiquibot/conversation-rewards
+- Service: Conversation Reward Module
+- Purpuse: This is intended to be the proper implementation of comment incentives, based on our learnings from the first go-around.
+- Language: TypeScript
+- External Integration: Github Action, Github API
+- Database: None
```

Comment:
@gentlementlegen is it safe to assume that we can write Cloudflare KV for all of these or can you clarify the correct answers?